website/build_site.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+cp -r ../documentation content/
+jupyter nbconvert ../source/Mlos.Notebooks/*.ipynb --to markdown --output-dir content/notebooks
+
+cp ../*.md content/
+cp ../LICENSE content/
+cp ../README.md content/_index.md
+mv content/documentation/README.md content/documentation/_index.md
+sed -i 's/\.md/\//g' content/*.md
+sed -i 's/\.md/\//g' content/documentation/*.md
```

Comment:
Thanks for this as well.

Maybe also include a comment regarding why we're doing the `sed` (e.g. to preserve relative links for browsing in github as well).

Reply:
Looks good.  Thanks!